### PR TITLE
disable dragstart event when droppable isnt set

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "fullcalendar",
   "title": "FullCalendar",
-  "version": "2.3.2-bcherny",
+  "version": "2.3.2.1-bcherny",
```
Comment (bcherny) [inline bower.json]: what does this syntax mean?

File: src/common/Grid.js [hunk 2]
```diff
@@ -337,13 +337,17 @@ var Grid = fc.Grid = RowRenderer.extend({
 
 	// Binds DOM handlers to elements that reside outside the grid, such as the document
 	bindGlobalHandlers: function() {
-		$(document).on('dragstart sortstart', this.externalDragStartProxy); // jqui
+		if (this.view.opt('droppable')) {
```
Comment (bcherny) [inline src/common/Grid.js:340]: can the option be changed at runtime? if so, do we rebind global handlers when that happens?

File: bower.json (hunk 1 above)
Comment (ghost) [inline bower.json]: Otherwise what version would you recommend?

File: src/common/Grid.js (hunk 2 above)
Comment (ghost) [inline src/common/Grid.js:340]: No it can't.  Do you want it to do that?

Comment (bcherny) [inline src/common/Grid.js:340]: just making sure we've thought about this case!

File: bower.json (hunk 1 above)
Comment (bcherny) [inline bower.json]: 2.3.3? what does 2.3.2.1 mean?

**Comment by bcherny** [comment]
:+1:

File: bower.json (hunk 1 above)
Comment (ghost) [inline bower.json]: It's really just a unique identifier to differentiate between your version (`2.3.2-bcherny`) and this one.

Comment (ghost) [inline bower.json]: OK, I'll use `2.3.3`.  My original rationale is that I didn't want to get version numbers mixed up between this fork and upstream.

**Pull request merged**